pages/search/advanced.tsx

```diff
@@ -197,68 +190,39 @@ export default function AdvancedSearch({ isAuthenticated }) {
-                  labelText="Format"
-                  onChange={handleCheckboxChange}
-                  value={searchFormState["filters"].materialType}
-                  __css={{
-                    "> div": {
-                      display: "grid",
-                      gridTemplateColumns: { md: "repeat(2, minmax(0, 1fr))" },
-                      gridGap: "var(--nypl-space-s)",
-                      div: {
-                        marginTop: "0 !important",
-                      },
-                    },
-                  }}
-                >
-                  {materialTypeOptions.map((materialType) => {
-                    return (
-                      <Checkbox
-                        id={materialType.value}
-                        key={materialType.value}
-                        labelText={materialType.label}
-                        value={materialType.value}
-                      />
-                    )
-                  })}
-                </CheckboxGroup>
-              </FormField>
-            </FormRow>
-          </FormField>
+              <FormField>{<DateForm {...dateFormProps} />}</FormField>
+            </Box>
+            <Flex direction="column" gap="l">
+              <SearchFilterCheckboxField
+                options={searchAggregations.buildingLocation}
+                name="location"
+                label="Location"
+                handleCheckboxChange={(e) =>
+                  handleCheckboxChange("buildingLocation", e)
+                }
+                searchFormState={searchFormState["filters"].buildingLocation}
+              />
+              <SearchFilterCheckboxField
+                options={materialTypeOptions}
+                name="format"
+                label="Format"
+                handleCheckboxChange={(e) =>
+                  handleCheckboxChange("materialType", e)
+                }
+                searchFormState={searchFormState["filters"].materialType}
+              />
+            </Flex>
+          </Flex>
           <HorizontalRule __css={{ margin: 0 }} />
-          <FormRow>
-            <FormField>
-              <ButtonGroup
-                id="advancedSearchButtons"
-                __css={{
-                  marginLeft: "auto",
-                }}
-              >
-                <Button
-                  type="button"
-                  id="advancedSearchClear"
-                  buttonType="secondary"
-                  onClick={handleClear}
-                  size="large"
-                >
-                  Clear
-                </Button>
-                <Button id="advancedSearchSubmit" type="submit" size="large">
-                  Submit
-                </Button>
-              </ButtonGroup>
-            </FormField>
-          </FormRow>
+          <Flex justifyContent="end">
+            <CancelSubmitButtonGroup
+              formName="advanced-search"
+              cancelHandler={handleClear}
+              cancelLabel="Clear Fields"
```

Comment:
You might wanna double-check with Apoorva if fields should be lowercase since we've been moving towards globally using sentence case.

Reply:
I just updated them all to be sentence case!